source/common/umf_pools/disjoint_pool.cpp

```diff
@@ -293,6 +293,15 @@ class DisjointPool::AllocImpl {
         // Generate buckets sized such as: 64, 96, 128, 192, ..., CutOff.
         // Powers of 2 and the value halfway between the powers of 2.
         auto Size1 = params.MinBucketSize;
+        // Special case for when MinBucketSize equals 1. In this case buckets
```

Comment:
but why is this special case here...? (pls describe at least in the commit message)

Reply:
Done.